ovirt/resource_ovirt_vm_template.go

```diff
@@ -83,6 +83,16 @@ func resourceOvirtTemplate() *schema.Resource {
 				},
 				Description: "in MB",
 			},
+			"guaranteed_memory": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				ValidateFunc: validation.IntAtLeast(1),
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
Can you explain why do we need this?

Reply:
actually, I'm not sure that it's needed, copy/paste syndrome :)  , I guess it used to enforce defaults when is not set. I'll remove it

Reply:
done

Reply:
sorry didnt   push that change, check again

Reply:
I still see the DiffSuppressFunc